kvdb-rocksdb/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
+
+## [0.6.0] - 2019-02-28
```

Comment:
Breaking because of license change? Or…?

Reply:
because of #347, adding a field to a public struct is a breaking change, because
```rust
Config { a, b }
```
won't compile anymore.